src/coreclr/nativeaot/Runtime/unix/PalCreateDump.cpp

```diff
@@ -13,10 +13,11 @@
 #include <unistd.h>
 #define __STDC_FORMAT_MACROS
 #include <inttypes.h>
+#include <signal.h>
```

Comment:
Presumably all changes in this file can now be reverted.

Reply:
yes, thanks